src/main/java/org/opentripplanner/client/OtpApiClient.java

```
@@ -143,6 +143,20 @@ public Stop stop(String gtfsId) throws IOException {
     return deserialize(jsonNode, "/data/stop", Stop.class);
   }
 
+  public List<Stop> stops(String nameMask) {
```

Please make this throw an IOException rather than returning an empty list.